docs/version0.33.md

```diff
@@ -22,9 +22,9 @@
 * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# What's new in version 0.33.0
+# What's new in version 0.33.0 and 0.33.1
```

Comment:
See comment for mkdocs.yml - suggest either just "0.33.1" or "0.33.x"

Reply:
Used 0.33.x